src/panels/logbook/ha-logbook.ts

```diff
@@ -198,14 +187,30 @@ export class HaLogbook extends LitElement {
     );
   }
 
+  private _unsubscribe(): void {
+    if (this._subscribed) {
+      this._subscribed.then((unsub) => unsub());
+      this._subscribed = undefined;
+    }
+  }
+
+  public disconnectedCallback() {
```

Comment:
if we add this, we should also add a `connectedCallback` or else we won't re-subscribe.

Reply:
09351ac7cade90c2f90784d59cb01476ab22d148
f55d7650634430021dc168b932b12fd87b06c960

Reply:
Need to go back to the history of the homeassistant websocket js api changes to remember how to test the disconnect/reconnect cycle

Reply:
https://github.com/home-assistant/home-assistant-js-websocket/pull/255

👌

Will test after sleep

Reply:
Added a testing disconnect button and hit it many times to make sure it reconnected ok 👍